package.json

```diff
@@ -19,24 +19,15 @@
     "body-parser": "^1.19.0",
     "cors": "^2.8.4",
     "dotenv": "^8.0.0",
-    "ejs": "^2.6.1",
-    "errorhandler": "^1.5.0",
     "express": "^4.17.1",
-    "express-jwt": "^5.3.1",
-    "express-session": "^1.15.6",
     "jsonwebtoken": "^8.3.0",
-    "method-override": "^2.3.10",
-    "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "path": "^0.12.7",
```

Comment:
Node has a path module. Unless there's a valid reason to install this from the npm registry, it should be removed as well

Reply:
yes, you are right. I've removed it. thanks